### DISCUSSION TITLE
Multidatabase environment

### DISCUSSION THREAD

**Discussion by jeroni** [mikro-orm/mikro-orm#5914] [Q&A]
I have an application using a multidatabase environment.

For each database, I create the corresponding ORM object.

Each request (based on the URL) registers the corresponding entity manager to the request context.

In some services, I have to consult another database for information. In these cases, I create a fork from another ORM. However, I must set disableContextResolution to use a connection to the other database.

Does this have any side effects? Must I manually return the connection to the pool? Is there a better way to achieve this?

**Comment by B4nan** [comment]
Sounds good to me, that's exactly what that option is for. I don't think you need to clean up any connections, that's all handled by knex, just don't keep a reference to the fork anywhere so it can be garbage collected.
